### PR TITLE
working on additional-text

### PR DESCRIPTION
**Pull request by SebasG22**
adding additional text to `index.html`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <head>
     <title> Register User</title>
+    <link rel="stylesheet" href="./index.css">
 </head>
 <body>
-<h1> Enter your information</h1>
+<h1 class="additional_text"> Enter your information</h1>
+<p> Fill the following form:</p>
```
Comment (stefymojica) [inline index.html]:
```suggestion
<p class="additional_text"> Fill the following form:</p>
```

File: index.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <head>
     <title> Register User</title>
+    <link rel="stylesheet" href="./index.css">
 </head>
 <body>
-<h1> Enter your information</h1>
+<h1 class="additional_text"> Enter your information</h1>
```
Comment (stefymojica) [inline index.html]:
```suggestion
<h1> Enter your information</h1>
```

**Review by stefymojica — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (SebasG22) [reply index.html]: Fixed in the last commit, please check.

File: index.html (hunk 1 above)
Comment (SebasG22) [reply index.html]: Fixed in the last commit, please check.

**Review by stefymojica — APPROVED** [review]

**Pull request merged**